_data/tool_and_resource_list.yml

```diff
@@ -15,580 +15,724 @@
   id: mypy
   name: mypy
   url: https://mypy-lang.org/
+  catalog: TechRadar
```

Comment:
OK, we've discussed this in today's coworking session, and the decision was:

We need to update the metadata guidelines to describe this field - this can be done separately as https://github.com/EVERSE-ResearchSoftware/RSQKit/issues/470
To merge this PR after logging this follow on work. (so I'm about to do that :-) )
Many thanks for the change!

The other question that remains is the naming of this - I note the original branch is called "source_field" which does make more sense to me personally than "catalog", but we agreed to merge and update if needed later.

Many thanks!